docs/terminals.md

```diff
@@ -58,7 +58,7 @@ you use `runc` directly in something like a `systemd` unit file. To disable
 this `LISTEN_FDS`-style passing just unset `LISTEN_FDS`.
 
 **Be very careful when passing file descriptors to a container process.** Due
-to some Linux kernel (mis)features, a container with access to certain types of
+to some Linux kernel misfeatures, a container with access to certain types of
```

Comment:
We need to fix it to pass the checking code spell.

Reply:
Doesn't seem relevant to the PR

Reply:
But CI won't allow this word... if I edit this file, CI won't go through unless I fix this part.

Reply:
Please remove this hunk. We'll deal with CI later.